backend/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthDto } from './dto/auth.dto';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('signup')
+  async signup(@Body() authDto: AuthDto): Promise<any> {
```

Comment:
don't use Promise<any> in controller, set proper type or just delete return type to make it automatical